client/UI/FFRKViewInventory.cs

```diff
@@ -116,6 +118,34 @@ private void FFRKViewInventory_Load(object sender, EventArgs e)
             }
         }
 
+        private void FFRKViewInventory_Recalc()
+        {
+            if (DesignMode)
+                return;
+
+            if (FFRKProxy.Instance != null)
+            {
+                mAnalyzerSettings = AnalyzerSettings.DefaultSettings;
+                mAnalyzerSettings.LevelConsideration = TranslateLevelConsideration((ScoreUpgradeModeComboIndex)comboBoxScoreSelection.SelectedIndex);
+                mAnalyzer = new EquipmentAnalyzer(mAnalyzerSettings);
+
+                FFRKProxy.Instance.OnPartyList += FFRKProxy_OnPartyList;

```

Comment:
Addressed this in the next update, thanks for the comments. I included it all because the fiddler inspector wasn't loading without the stuff, but apparently all I needed was "if (party != null) { UpdateEquipmentGrid(party.Equipments); }" (when I removed this if statement everything broke).